README.md

```diff
@@ -43,6 +43,8 @@ Math magicians is a website for all fans of mathematics
 
 ### Key Features <a name="key-features"></a>
 
+- **Calculator**
+
```

Comment:
### FUTURE FEATURES


- You have submitted a project with an empty "FUTURE FEATURES" section. Kindly add some future features about your project to make your README more professional +1 

<img alt="Screenshot 2023-03-20 at 8 56 12 PM" width="851" src="https://user-images.githubusercontent.com/25887253/226426528-7e30c4dd-6e04-4602-97da-395aa8aba6a2.png">


- All your `back-to-top` links in the readme is not working, ensure you fix this issue so they link to the corresponding top navigation section of the readme file.